File: Assets/AltUnityTester/AltUnityDriver/Commands/UnityCommands/LoadScene.cs
public class LoadScene : AltBaseCommand
{
    string sceneName;
    public LoadScene(SocketSettings socketSettings,string sceneName) : base(socketSettings)
    {
        this.sceneName=sceneName;
    }
    public void Execute()
    {
        Socket.Client.Send(toBytes(CreateCommand("loadScene", sceneName)));
        var data = Recvall();
        if (data.Equals("Ok"))
        {
            data = Recvall();
            if(data.Equals("Scene Loaded"))
                return;
        }
        HandleErrors(data);
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/UnityCommands/load_scene.py
from altunityrunner.commands.base_command import BaseCommand
class LoadScene(BaseCommand):
    def __init__(self, socket,request_separator,request_end,scene_name):
        super(LoadScene, self).__init__(socket,request_separator,request_end)
        self.scene_name=scene_name
    
    def execute(self):
        data = self.send_data(self.create_command('loadScene', self.scene_name))
        if (data == 'Ok'):
            data=self.recvall()
            if (data =="Scene Loaded"):
                print('Scene loaded: ' + self.scene_name)
                return data
        return None